hal/src/boron/network/sara_ncp_client.cpp

```diff
@@ -155,46 +155,86 @@ int SaraNcpClient::initParser(Stream* stream) {
             .commandTerminator(AtCommandTerminator::CRLF);
     parser_.destroy();
     CHECK(parser_.init(std::move(parserConf)));
+    // +CREG: <stat>[,<lac>,<ci>[,<AcTStatus>]]
     CHECK(parser_.addUrcHandler("+CREG", [](AtResponseReader* reader, const char* prefix, void* data) -> int {
         const auto self = (SaraNcpClient*)data;
-        int val[2];
-        int r = CHECK_PARSER_URC(reader->scanf("+CREG: %d,%d", &val[0], &val[1]));
+        int val[4] = {-1,-1,-1,-1};
+        char at_response[64] = {0};
```

Comment:
`snake_case` in `camelCase` file, `0` is unnecessary, `{}` is enough. I think this is the third time I'm complaining about `snake_case` in exactly the same place :)

Reply:
You finally pierced my skull! `snake_case` is now `camelCase`!